src/block/VanillaBlocks.php

```
@@ -113,6 +113,7 @@
  * @method static FloorBanner BANNER()
  * @method static Barrel BARREL()
  * @method static Transparent BARRIER()
+ * @method static Transparent STRUCTURE_VOID name()
```

PHPStan (2.1.15)

PHPDoc tag @method has invalid value (static Transparent STRUCTURE_VOID name()): Unexpected token "name", expected '(' at offset 1724 on line 41


(phpDoc.parseError)